pyproject.toml

```diff
@@ -15,7 +15,7 @@ dependencies = [
     "attrs>=21.3",
     "build>=0.7.0",
     "cachetools>=4.1",
-    "click>=4.0,<8.2.0",
+    "click>=4.0",
```

Comment:
Not sure if these changes still remain backwards compatible, maybe we have to pin it to >=8.2.0